readme/otherattributemodifier/bool.go

```diff
@@ -0,0 +1,258 @@
+package otherattributemodifier
```

Comment:
New package to enable triggering a change of an attribute value when another attribute value changes. These are needed on some of the unstable attributes provided by the ReadMe API, such as `updated_at` and `revision`.

These enable those attributes to get updated at the same time as the 'other' attribute. This addresses a bug that caused inconsistent state after applying changes in some resources (e.g. docs).